app/src/main/java/com/arctouch/codechallenge/extensions/DateExtensions.kt

```diff
@@ -0,0 +1,30 @@
+package com.arctouch.codechallenge.extensions
+
+import java.text.ParseException
+import java.text.SimpleDateFormat
+import java.util.*
+
+@JvmOverloads
+fun String.convertToFormattedDate(language: String? = null): String{
```

Comment:
What was your motivation for using Kotlin to create these extension functions?
Did you think about using Kotlin for other parts of the app?

Reply:
My motivation was, as this project contains Kotlin support and thinking in the future that this project could be completely converted to Koltin, this made sense to me. Yes, I thought about using Kotlin for the entire app actually, but I don’t have too much knowledge on it yet, so that would be risky. One thing that could be done is converting parts of the app, not everything, that’s one of the benefits of using Kotlin which is the easy way of communication with Java.